src/Client/ClientBase.cpp

```diff
@@ -122,6 +123,7 @@ namespace ProfileEvents
 namespace
 {
 constexpr UInt64 THREAD_GROUP_ID = 0;
+const std::string UNICODE_DASH = "—";
```

Comment:
We should support en-dash, em-dash, and mathematical minus.

Reply:
No need for the declaration of constant; it's ok to put these constants inline in the code.

Reply:
fixed
Added support for en-dash and mathematical minus